apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt

```diff
@@ -226,10 +263,26 @@ private fun ChatListTopBar(stopped: Boolean) {
       }
     }
   }
-
+  val scope = rememberCoroutineScope()
   val clipboard = LocalClipboardManager.current
-
   DefaultTopAppBar(
+    navigationButton = {
+      if (chatModel.users.isEmpty() && !chatModel.desktopNoUserNoRemote) {
+        NavigationButtonMenu { scope.launch { if (drawerState.isOpen) drawerState.close() else drawerState.open() } }
+      } else {
+        val users by remember { derivedStateOf { chatModel.users.filter { u -> u.user.activeUser || !u.user.hidden } } }
+        val allRead = users
+          .filter { u -> !u.user.activeUser && !u.user.hidden }
+          .all { u -> u.unreadCount == 0 }
+        UserProfileButton(chatModel.currentUser.value?.profile?.image, allRead) {
```

Comment:
in regular mode (one hand UI off) user picker is incorrectly located in the bottom

Reply:
fixed